hudi-common/src/main/java/org/apache/hudi/common/table/log/BaseHoodieLogRecordReader.java

```diff
@@ -260,7 +260,7 @@ private void scanInternalV1(Option<KeySpec> keySpecOpt) {
             && !HoodieTimeline.compareTimestamps(logBlock.getLogBlockHeader().get(INSTANT_TIME), HoodieTimeline.LESSER_THAN_OR_EQUALS, this.latestInstantTime
         )) {
           // hit a block with instant time greater than should be processed, stop processing further
-          break;
+          continue;
```

Comment:
hey folks. do we need this fix for 0.14.x as well. or do we need only for 1.x? 
in other words, should I cherry this for 0.14.1 upcoming relase?